### DISCUSSION TITLE
微信群二维码, 不定期更新

### DISCUSSION THREAD

**Discussion by lxzan** [lxzan/gws#30] [Announcements]
<img src="https://github.com/lxzan/gws/assets/17956540/5466ece1-1bdf-4360-865f-fb85cc4b01a4" width=300>)

**Comment by houseme** [comment]
过期了，从 QQ 群追到这里哈 @lxzan 来更新二维码哈

**Comment by Faker0715** [comment]
大佬 过期了 @lxzan

**Reply by lxzan** [reply]
更新了

**Reply by vvair** [reply]
想进群，能否麻烦更新一下码

**Reply by lxzan** [reply]
> 想进群，能否麻烦更新一下码

更新了
